app/views/shared/_reference_plans_list.html.erb

```diff
@@ -109,7 +109,11 @@
       <br><br>
       <p class="twenty">Select your preferred metal level. Your plan participants will be able to choose any plan by any carrier within the metal level you select. Your costs will be fixed to a specific plan you’ll choose in a minute. Bronze means the plan is expected to pay 60% of expenses for an average population of consumers, Silver 70%, Gold 80% and Platinum 90%.</p>
       <br/>
-      <% Plan::REFERENCE_PLAN_METAL_LEVELS.each do |kind| %>
+      <% reference_plans = Plan::REFERENCE_PLAN_METAL_LEVELS %>
```

Comment:
does this show dental as an option when choosing a reference health plans metal level? 

Is this desired?